### PR TITLE
Simple api2 run scenario fix

### PR DESCRIPTION
**Pull request by kklocek**
Previously we have some bugs and problems with running uploaded scenario:
* there was no information that scenario is not started
* scenario crashes if it uses lager which need special compile flag during compilation
* if we compiled scenario then we couldn't load it because in one line we provide directory name as atom, not a string and we couldn't use scenario later.

### CODE REVIEW

File: src/amoc_api_scenario_handler.erl [hunk 1]
```diff
@@ -173,3 +175,14 @@ get_users_from_body(Req) ->
               {error, bad_request, Req2}
     end.
 
+-spec get_result([ok | {error, term()}]) -> started | error.
+get_result(Result) ->
+    Res = lists:all(fun(X) -> X == ok end, Result),
+    case Res of
+        true -> started;
+        false ->
+                %% Internal server error, log problems
```
Comment (arkgil) [inline src/amoc_api_scenario_handler.erl]: What does 'log problems' refer to?

**Comment by arkgil** [comment]
Looks good to me.

**Comment by michalwski** [comment]
Could you please describe the bug shortly in the PR desc?

File: src/amoc.app.src [hunk 2]
```diff
@@ -7,6 +7,7 @@
                   kernel,
                   stdlib,
                   lager,
+                  jiffy,
```
Comment (michalwski) [inline src/amoc.app.src]: I know we need `jiffy` because of `trails`. We also have `jsx` which does pretty much the same as `jsx`. Could you check if we can replace `jsx` with `jiffy`? If so, then we should do this in a separate PR.

**Comment by kklocek** [comment]
Added bugs description.

File: src/amoc.app.src (hunk 2 above)
Comment (kklocek) [inline src/amoc.app.src]: We also need a `jiffy` for `cowboy-swagger` .
So, the question is to replace `jsx` with `jiffy`? I will try.
